deploy/html/user_secondfactor_add.html

```diff
@@ -0,0 +1,18 @@
+{%- extends "viur_base.html" %}
+
+{%- block content %}
+<h1 class="headline">Add Second Factor</h1>
+	<p>Please scan the QR code with your Authenticator App and Enter the OTP token in the input below</p>
+		{{otp_uri_to_qrcode(otp_uri)}}
```

Comment:
Does this generate a SVG HTML Element or an link/base64 to an image and should be a an image and clickable html link?

```suggestion
		{%- set uri = otp_uri_to_qrcode(otp_uri) -%}
		<a href="{{ uri }}"><img src="{{ uri }}"</a>
```